source/common/upstream/default_local_address_selector.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "envoy/upstream/upstream.h"
+
+#include "absl/types/optional.h"
+
+namespace Envoy {
+namespace Upstream {
+
+/**
+ * Default implementation of UpstreamLocalAddressSelector.
```

Comment:
nit: consider adding some explanation on what it does, or link to the relevant docs where this is explained.